frontend/.nyc_output/out.json

```diff
@@ -0,0 +1,3985 @@
+{
+  "/Users/marikoanderson/fe_v3/src/components/auth/Login.jsx": {
```

Comment:
this file has path from your user, which makes me wonder if this directory something that needs to be checked in?

Reply:
I've added this to git ignore because it is and auto generated code coverage report from cypress

Reply:
Added to git ignore as you suggested.